.github/dependabot.yml

```diff
@@ -5,20 +5,51 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
-      time: "09:00"
-      timezone: "Europe/London"
-    commit-message:
-      prefix: "fix:"
-      prefix-development: "chore:"
```

Comment:
Is the removal of this right? I assumed these were here because of commitlint/release-please

Reply:
Oops! That's very annoying of me

Reply:
I'm going to add it to all of them. How irritating 😂

Reply:
Sorry!